faq.rst

```diff
@@ -112,9 +112,66 @@ explicitly with ::
 
 https://en.wikipedia.org/wiki/EGL_(API)
 
+How to achieve transparency with 2D objects?
+--------------------------------------------
+
+With objects that lie in a 2D plane, e.g. images, it is possible to get a
+translucent effect (i.e. partial transparency, a see-through effect) by
+positioning them at different depth levels in the viewing direction and by
+drawing the visuals from furthest to closest with the appropriate blending
+mode.
+
+Here are the key steps to achieve this with two
+:class:`~vispy.scene.visuals.Image` visual nodes in a
+:class:`~vispy.scene.canvas.SceneCanvas`:
+
+1. Set the opacity value in the alpha channel of the images::
+  
+    # A white image with integer values between 0 and 255.
+    image_data1 = np.ones((200, 300, 4), dtype='uint8')
+    # Half translucent.
+    image_data1[..., 3] = 128
+
+    # A blue image with float values between 0 and 1.
+    image_data2 = np.zeros((200, 300, 4), dtype='np.float32')
+    image_data2[..., 2]  = 1.0  # Blue.
+    # A bit more translucent.
+    image_data2[..., 3] = 0.25
+
+    visual1 = Image(image_data1)
+    visual2 = Image(image_data2)
+
+2. Position the visuals at different depth levels (z-levels) in the viewing
+   direction: ::
```

Comment:
```suggestion
   direction::
```

Could you fix all of these cases of double colons?

Reply:
Thanks @djhoese . Fixed. I didn't know the difference between `::` and ` ::`.

Reply:
I don't think there is an overall difference but I guess I'm just being picky as `: ::` isn't necessary.